search/src/Components/error/utils.js

```diff
@@ -0,0 +1,17 @@
+export const jsonSchemaTitle = (res) => {
+    return `Total ${res.n_errors_found} ${res.n_errors_found > 1? "errors": "error"}`
+}
+
+export const jsonSchemaTransform = (details) => {
+    const transform = {}
+    const pathKey = (path) => `'${path[2]}' field of ${path[0]}`
```

Comment:
This won't behave appropriately if:
* the error is in `info`, since it is not an array
* the bad field is deeper, such as `annotations.0.segmentation.counts`

So you might want to do something like:
```js
const namedPaths = path.filter(word => typeof word == typeof '');
```
For pathKey you might want to leave it as `namedPaths.join('.')` but for display you might want to switch based on path length such that:
* ['categories'] -> categories
* ['categories', 'name'] -> name field of categories
* ['annotations', 'segmentation', 'counts'] -> counts field of segmentation in annotations

or don't bother with that level of intricacy initially and just display namedPaths.join('.') when it's not of length 2.

I'd probably not transform for the purpose of display until as late as possible, since you might want to format those path elements. Constructing a string here means that you can't format it as react components later.

Deserves unit tests.